include/sys/fs/zfs.h

```diff
@@ -799,13 +799,15 @@ typedef struct pool_scan_stat {
 	uint64_t	pss_start_time;	/* scan start time */
 	uint64_t	pss_end_time;	/* scan end time */
 	uint64_t	pss_to_examine;	/* total bytes to scan */
-	uint64_t	pss_examined;	/* total examined bytes	*/
+	uint64_t	pss_examined;	/* total bytes located by scanner */
+	uint64_t	pss_issued;	/* total bytes checked by scanner */
```

Comment:
On Linux I'm pretty sure we just deal with this by printing an error message saying that the binary version doesn't match the kernel version and exitting. @behlendorf can you confirm this?